package.json

```diff
@@ -1641,6 +1641,18 @@
         "category": "Calva Refactor",
         "enablement": "editorLangId == clojure && clojureLsp:active"
       },
+      {
+        "command": "calva.refactor.dragBackward",
+        "title": "Drag Sexp Backward",
+        "category": "Calva Refactor",
+        "enablement": "editorLangId == clojure && clojureLsp:active"
+      },
+      {
+        "command": "calva.refactor.dragForward",
+        "title": "Drag Sexp Forward",
+        "category": "Calva Refactor",
+        "enablement": "editorLangId == clojure && clojureLsp:active"
```

Comment:
You've probably thought of this too. These are not refactors, in any meaningful sense of the word. We should name these something else. Maybe Instead of ”Calva Refactor”, we can use "clojure-lsp". That should make the option clearer about which system is handling it.

While at it we could recategorize the other ”Calva Refactor” commands to say ”clojure-lsp Refactor”.

Reply:
I like it